core/eolearn/tests/test_core_tasks.py

```diff
@@ -342,38 +342,33 @@ def test_zip_features_fails(patch: EOPatch) -> None:
         ZipFeatureTask({FeatureType.DATA: ["CLP", "bands"]}, (FeatureType.DATA, "MERGED"))(patch)
 
 
-def test_map_features(test_eopatch):
-    move = MapFeatureTask(
-        {FeatureType.DATA: ["CLP", "NDVI", "BANDS-S2-L1C"]},
-        {FeatureType.DATA: ["CLP2", "NDVI2", "BANDS-S2-L1C2"]},
-        copy.deepcopy,
-    )
-
-    patch = move(test_eopatch)
-
-    assert np.array_equal(patch.data["CLP"], patch.data["CLP2"])
-    assert np.array_equal(patch.data["NDVI"], patch.data["NDVI2"])
-    assert np.array_equal(patch.data["BANDS-S2-L1C"], patch.data["BANDS-S2-L1C2"])
-
-    assert id(patch.data["CLP"]) != id(patch.data["CLP2"])
-    assert id(patch.data["NDVI"]) != id(patch.data["NDVI2"])
-    assert id(patch.data["BANDS-S2-L1C"]) != id(patch.data["BANDS-S2-L1C2"])
-
-    map_fail = MapFeatureTask(
-        {FeatureType.DATA: ["CLP", "NDVI"]},
-        {
-            FeatureType.DATA: [
-                "CLP2",
-                "NDVI2",
-            ]
-        },
-    )
+@pytest.mark.parametrize(
+    "feat_spc_org, feat_spc_map, fun",
+    [
+        ({FeatureType.DATA: ["CLP", "bands"]}, {FeatureType.DATA: ["CLP_max", "bands_max"]}, lambda x: x + 3),
+        ({FeatureType.MASK_TIMELESS: ["mask", "LULC"]}, {FeatureType.MASK_TIMELESS: ["mask2", "LULC2"]}, copy.deepcopy),
+        ({FeatureType.DATA: ["CLP", "CLP_S2C"]}, {FeatureType.DATA: ["CLP_round", "CLP_S2C_round"]}, np.ceil),
+    ],
```

Comment:
there is no test case where the new feature is the old feature (overwriting) 

can be done in separate test if desired

Reply:
Add separate test.